src/effects/EffectUI.cpp

```diff
@@ -309,15 +309,24 @@ int EffectUIHost::ShowModal()
    mApplyBtn->Destroy();
    mApplyBtn = apply;
    mApplyBtn->SetDefault();
-   mApplyBtn->SetLabel(wxGetStockLabel(wxID_OK, 0));
+
+   if (mIsBatch)
+      mApplyBtn->SetLabel(wxGetStockLabel(wxID_OK, 0));
+   else if (mEffectUIHost.GetDefinition().GetType() == EffectTypeGenerate)
+      mApplyBtn->SetLabel(XO("Generate").Translation());
```

Comment:
I would like to see `XXO("&Generate")`. This way this string will be reused.

Reply:
Done.